app/celery/tasks.py

```diff
@@ -335,29 +329,6 @@ def save_letter(
         handle_exception(self, notification, notification_id, e)
 
 
-@notify_celery.task(bind=True, name="build-dvla-file", countdown=60, max_retries=15, default_retry_delay=300)
-@statsd(namespace="tasks")
-def build_dvla_file(self, job_id):
-    try:
-        if all_notifications_are_created_for_job(job_id):
-            file_contents = create_dvla_file_contents_for_job(job_id)
-            s3upload(
-                filedata=file_contents + '\n',
-                region=current_app.config['AWS_REGION'],
-                bucket_name=current_app.config['DVLA_BUCKETS']['job'],
-                file_location="{}-dvla-job.text".format(job_id)
-            )
-            dao_update_job_status(job_id, JOB_STATUS_READY_TO_SEND)
-        else:
-            msg = "All notifications for job {} are not persisted".format(job_id)
-            current_app.logger.info(msg)
-            self.retry(queue=QueueNames.RETRY)
-    # specifically don't catch celery.retry errors
-    except (SQLAlchemyError, BotoClientError):
-        current_app.logger.exception("build_dvla_file threw exception")
-        self.retry(queue=QueueNames.RETRY)
-
-
 @notify_celery.task(bind=True, name='update-letter-job-to-sent')
 @statsd(namespace="tasks")
 def update_job_to_sent_to_dvla(self, job_id):
```

Comment:
Is this also deprecated as letter pdfs are processed by date rather than job

Reply:
good spot